### DISCUSSION TITLE
When I choose Raven cloud, do I have then to buy the license too ?

### DISCUSSION THREAD

**Discussion by pullmann4rent** [ravendb/ravendb#18024] [Q&A] [answered]
Hello,

When I want to use raven cloud and I purchase it, do I have also buy the license ? If yes how much costs the pro license then ? And what are my disadvantages when I use the free license ?

**Comment by DejanMilicic** [comment, marked as answer]
RavenDB Cloud comes with enterprise-grade license, so you do not need to make any additional purchases.
Regarding free license, are you asking about free cloud instance or Community license?
